src/store/utils.js

```diff
@@ -38,9 +40,34 @@ export const retryIfSourceChainHeadMoved = async call => {
     console.log('isHeadMovedError', isHeadMovedError)
     if (isHeadMovedError) {
       intervalMs *= (2 + Math.random())
-      await delay(intervalMs)
```

Comment:
@timotree3 - `delay` is never defined, so this path would just throw an error.

Reply:
oops! good catch